kernelci/rootfs.py

```diff
@@ -44,17 +43,25 @@ def _build_debos(name, config, data_path):
             test_overlay=config.test_overlay,
             crush_image_options=" ".join(config.crush_image_options)
             )
-        shell_cmd(cmd)
+    shell_cmd(cmd)
 
     return True
```

Comment:
Note that this return value is not used anywhere.  But this is fine as any error would cause an exception before reaching this line anyway.